test/fixtures.ts

```diff
@@ -0,0 +1,77 @@
+import type fs from 'fs';
+import type { PathLike } from 'fs';
+import callsite from 'callsite';
+import { DirectoryJSON, fs as memfs, vol } from 'memfs';
+import upath from 'upath';
+
+export class Fixtures {
+  static get(name: string, fixturesRoot = '.'): string {
```

Comment:
Please add js-doc comments to functions, so everybody knows what they are doing.

Reply:
Added, let me know if need more information